locales/no-NO.yml

```
@@ -0,0 +1,4 @@
+"no":
```

🔧 I believe this should be
```suggestion
no:
```